src/models/ObjectLevelCan.ts

```diff
@@ -1,17 +1,21 @@
-type BasicPermissions = 'create' | 'read' | 'update' | 'delete'
-
-
 type ObjectLevelCan<TPermissions extends string> = {
   [key in TPermissions]: boolean
 }
 
-export type BasicPermissionsObjectLevelCan = ObjectLevelCan<BasicPermissions>
-export type BasicRunnablePermissionsObjectLevelCan = ObjectLevelCan<BasicPermissions | 'run'>
+export const BasicObjectLevelPermissions = ['create', 'read', 'update', 'delete'] as const
+export const RunnableObjectLevelPermissions = [...BasicObjectLevelPermissions, 'run'] as const
```

Comment:
I want to tie these to the actual permissions so we get type errors and such when permissions are being changed. Tried a few things but the best idea I came up with was to update `can.ts` like this.

```typescript
export const permissionVerbs = ['create', 'delete', 'read', 'run', 'update'] as const
export type PermissionVerbs = typeof permissionVerbs[number]

export const workspacePermissions = [
  'create:artifact',
  'create:automation',
  'create:block',
  'create:concurrency_limit',
  'create:deployment',
  'create:flow_run',
  'create:flow',
  'create:log',
  ...etc
] as const satisfies Readonly<`${PermissionVerbs}:${string}`[]>
```

Then use `permissionVerbs` and `PermissionVerbs` in `createObjectLevelCan` as needed.